.gitignore

```diff
@@ -1,3 +1,5 @@
 # PyCache files
 __pycache__/
-cutlass_library.egg-info/
\ No newline at end of file
+cutlass_library.egg-info/
+build/
+*.patch
```

Comment:
Rather than adding extra folders in the .gitignore like build and for patch files, a better approach would be to include these in your global .gitignore config, this way we can keep the changes minimal.